docs/eventing/channels/channels-crds.md

```diff
@@ -16,10 +16,9 @@ This is a non-exhaustive list of the available Channels for Knative Eventing.
 !!! note
     Inclusion in this list is not an endorsement, nor does it imply any level of support.
 
-Name | Status | Support | Description
---- | --- | --- | ---
+Name | Status           | Support | Description
+--- |------------------| --- | ---
 [GCP PubSub](https://github.com/google/knative-gcp) | Proof of Concept | None | Channels are backed by [GCP PubSub](https://cloud.google.com/pubsub/).
 [InMemoryChannel](https://github.com/knative/eventing/tree/{{ branch }}/config/channels/in-memory-channel/README.md) | Proof of Concept | None | In-memory channels are a best effort Channel. They should NOT be used in Production. They are useful for development.
-[KafkaChannel - Consolidated](https://github.com/knative-sandbox/eventing-kafka/tree/{{ branch }}/pkg/channel/consolidated/README.md) | Proof of Concept | None | Channels are backed by [Apache Kafka](http://kafka.apache.org/) topics. The original Knative KafkaChannel implementation which utilizes a single combined Kafka Producer / Consumer deployment.
-[KafkaChannel - Distributed](https://github.com/knative-sandbox/eventing-kafka/tree/{{ branch }}/pkg/channel/distributed/README.md) | Proof of Concept | None | Channels are backed by [Apache Kafka](http://kafka.apache.org/) topics. An alternate KafkaChannel implementation, contributed by SAP's [Kyma](https://kyma-project.io/) project, which provides a more granular deployment of Producers / Consumers.
+[KafkaChannel](https://github.com/knative-sandbox/eventing-kafka-broker/tree/{{ branch }}/README.md) | Beta             | None | Channels are backed by [Apache Kafka](http://kafka.apache.org/) topics.
```

Comment:
I created https://github.com/knative/docs/pull/5001 for this line. Linking to https://github.com/knative-sandbox/eventing-kafka-broker/blob/main/docs/channel/README.md